lib/python/rosie/browser/main.py

```diff
@@ -798,19 +798,17 @@ def _handle_display_change(self, menuitem):
         self.display_toggle(menuitem.column)
 
     def _handle_prefix_change(self, menuitem):
-        """Handles changing the prefix."""
-        prefixes = list(self.ws_client.prefixes)
-        old_prefixes = list(prefixes)
-        if menuitem.get_active():
-            if menuitem.prefix_text not in prefixes:
-                prefixes.append(menuitem.prefix_text)
-                prefixes.sort()
```

Comment:
From ws_client.set_prefixes():
```
if self.prefixes != prefixes:
    self.prefixes = prefixes
```
As `[1, 2, 3] != [3, 2, 1]` the code can execute if the prefixes haven't changed but are provided in a different order. I can't see how this would effect anything, we could move the sort to set_prefixes() to be safe.